src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 #========================================
 # General settings
 #========================================
-spring.application.name=template
+spring.application.name=api-feedbacksettings-v2
```

Comment:
Should we rename (as discussed in Slack)?

Reply:
I'm on it!

Reply:
Changed service name to: api-service-contactsettings